spec/factories/search_fields.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :search_field do
+    
```

Comment:
And before we merge this branch, we have to wait @JohnnyJ0622 to finished data and test it.